apps/core-app/src/components/FakeMarkdown.tsx

```diff
@@ -0,0 +1,145 @@
+import { useEffect, useState } from 'react';
+
+import {
+  Buildable,
+  ErrorMessage,
+  Field,
+  SuccessMessage,
+  WrappedTextArea,
+} from '@daohaus/ui';
+import { useDao } from '../contexts/DaoContext';
+import { useParams } from 'react-router-dom';
+import {
+  ENDPOINTS,
+  isValidNetwork,
+  ReactSetter,
+  ValidNetwork,
+} from '@daohaus/common-utilities';
+
+import { gql, request } from 'graphql-request';
+import { useFormBuilder } from '@daohaus/haus-form-builder';
+import { FieldValues, UseFormSetValue } from 'react-hook-form';
+
+const PUB_ID = 'pubId';
+enum FetchStatus {
```

Comment:
Use a State Machine?
I'm not 100% clear on the logical flow from status to status and a state machine might not work here.